components/Device/widgets/RewardsWidgets/MainnetBanner.vue

```diff
@@ -2,41 +2,39 @@
   import { useTheme, useDisplay } from 'vuetify'
   import bannerBackground from '~/assets/bannerBackground.jpeg'
 
-  interface Props {
-    text?: string
-  }
-
-  const props = withDefaults(defineProps<Props>(), {
-    text: ''
-  })
-
   const theme = useTheme()
   const display = useDisplay()
   const smBreakpoing = computed(() => {
     return display.smAndDown
   })
 
-  console.log()
+  const { fetchRemoteConfig } = useFirebase()
+  const remoteConfig = await fetchRemoteConfig()
+  const mainnetBannerText = ref<string>(`${remoteConfig.feat_mainnet_message._value}`)
+  const mainnetUrl = ref<string>(`${remoteConfig.feat_mainnet_url._value}`)
+  const computedBannerText = computed(() => {
```

Comment:
If you don't want to use this opinionated solution that replaces characters in the original string, you can just add `white-space` properties in the banner text div like this:
```
<div style="white-space: pre-wrap" v-html="mainnetBannerText" />
```

Reply:
The disadvantage of using white-space: pre-wrap would be not accounting for additional spaces/tabs. In our case we're using \n\n in order to separate the last sentence.